spring-cloud-contract-wiremock/src/main/java/org/springframework/cloud/contract/wiremock/WireMockConfiguration.java

```diff
@@ -97,9 +97,6 @@ private void registerStubs() throws IOException {
 						this.resourceLoader);
 				String pattern = stubs;
 				if (!pattern.contains("*")) {
-					if (!pattern.endsWith("/")) {
```

Comment:
My issue was primarily with the registerFiles(), since I then could rely on the stubs being picked up from the <filesRoot>/mappings.  If you specify a subs location without the *.json then that code that adds the "/**/*.json" is helpful to find all of the mapping .json files.